src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/barnetilsyn/beregning/TilsynBarnVedtaksperiodeValideringUtils.kt

```diff
@@ -132,4 +132,39 @@ object TilsynBarnVedtaksperiodeValideringUtils {
             }
         }
     }
+
+    fun validerIngenEndringerFørRevurderFra(
+        vedtaksperioder: List<Vedtaksperiode>,
+        vedtaksperioderForrigeBehandling: List<Vedtaksperiode>?,
+        revurderFra: LocalDate?,
+    ) {
+        if (revurderFra == null) return
+
+        val vedtaksperioderFørRevurderFra = vedtaksperioder.filter { it.fom < revurderFra }
+        val vedtaksperioderForrigeBehandlingFørRevurderFra =
+            vedtaksperioderForrigeBehandling?.filter { it.fom < revurderFra }
+        val vedtaksperioderMap = vedtaksperioderFørRevurderFra.associateBy { it.id }
+
+        if (vedtaksperioderForrigeBehandlingFørRevurderFra.isNullOrEmpty()) {
+            brukerfeilHvis(vedtaksperioder.any { it.fom < revurderFra }) {
+                "Det er ikke tillat å legge til nye perioder før revurder fra dato"
+            }
+        } else {
+            val vedtaksperioderForrigeBehandlingFørRevurderFraMedOppdatertTom =
+                vedtaksperioderForrigeBehandlingFørRevurderFra.map { vedtaksperiodeForrigeBehandling ->
+                    val tilhørendeNyVedtaksperiode = vedtaksperioderMap[vedtaksperiodeForrigeBehandling.id]
+
+                    if (tilhørendeNyVedtaksperiode != null &&
+                        tilhørendeNyVedtaksperiode.tom > vedtaksperiodeForrigeBehandling.tom
+                    ) {
+                        vedtaksperiodeForrigeBehandling.copy(tom = tilhørendeNyVedtaksperiode.tom)
```

Comment:
Tror denne delen blir litt enklere å lese med litt kortere navn som gjør at if-sjekken ikke går over 2 rader 
```suggestion
                    val nyVedtaksperiode = vedtaksperioderMap[vedtaksperiodeForrigeBehandling.id]

                    if (nyVedtaksperiode != null && nyVedtaksperiode.tom > vedtaksperiodeForrigeBehandling.tom) {
                        vedtaksperiodeForrigeBehandling.copy(tom = nyVedtaksperiode.tom)
```

Reply:
Det kan vi gjøre!